### ISSUE TITLE
[Accesibility] Footer background and footer heading color have low contrast ratio

### ISSUE THREAD

**Issue by xNyfPtx** [NixOS/nixos-homepage#1754]
**Describe the glitch**
Not really a glitch but an accesibility issue. The background of the footer and the color of the footer headings have a low contrast ratio of 2.64:1 which fails WCAG AA and WCAG AAA guidelines.

**Screenshots**

![Image](https://github.com/user-attachments/assets/436e7af1-93ce-4ac8-9a26-29858dadaf30)
![Image](https://github.com/user-attachments/assets/f9f61d30-e1fc-41ab-a37b-f93b5ea7bfc4)
![Image](https://github.com/user-attachments/assets/854d2205-df6a-4009-80fb-421eda4eb5c0)
![Image](https://github.com/user-attachments/assets/bda5f8ac-373e-4b7c-a878-5106d98d510a)

**Guidelines**

[https://developer.mozilla.org/en-US/docs/Web/Accessibility/Guides/Understanding_WCAG/Perceivable/Color_contrast](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Guides/Understanding_WCAG/Perceivable/Color_contrast)
[https://www.w3.org/TR/WCAG21/#contrast-minimum](https://www.w3.org/TR/WCAG21/#contrast-minimum)
[https://www.w3.org/TR/WCAG21/#contrast-enhanced](https://www.w3.org/TR/WCAG21/#contrast-enhanced)
[https://www.w3.org/TR/WCAG21/#non-text-contrast](https://www.w3.org/TR/WCAG21/#non-text-contrast)

**Comment by thilobillerbeck** [comment]
Yup, that has been ported over from the old tech stack and is already on my radar, but thanks for bringing this up, we'll have that sorted soon :)

**Comment by thilobillerbeck** [comment]
Contrast is a 5.54:1 now, which should hopefully suffice for the headline sized text.
